src/bindings/python/fluxacct/accounting/jobs_table_subcommands.py

```diff
@@ -123,41 +124,32 @@ def write_records_to_file(job_records, output_file):
             )
 
 
-def convert_to_str(job_records):
+def convert_to_str(job_records, fields):
     """
     Convert the results of a query to the jobs table to a readable string
     that can either be output to stdout or written to a file.
     """
-    job_record_str = []
-    job_record_str.append(
-        "{:<10} {:<10} {:<20} {:<20} {:<20} {:<20} {:<10} {:<20} {:<20}".format(
-            "UserID",
-            "Username",
-            "JobID",
-            "T_Submit",
-            "T_Run",
-            "T_Inactive",
-            "Nodes",
-            "Project",
-            "Bank",
-        )
-    )
-    for record in job_records:
-        job_record_str.append(
-            "{:<10} {:<10} {:<20} {:<20} {:<20} {:<20} {:<10} {:<20} {:<20}".format(
-                record.userid,
-                record.username,
-                record.jobid,
-                record.t_submit,
-                record.t_run,
-                record.t_inactive,
-                record.nnodes,
-                record.project,
-                record.bank,
-            )
+    default_fields = [
+        "userid",
+        "username",
+        "jobid",
+        "t_submit",
+        "t_run",
+        "t_inactive",
+        "nnodes",
+        "project",
+        "bank",
+    ]
+    fields = fields or default_fields
+    invalid_fields = [field for field in fields if field not in default_fields]
```

Comment:
If I'm reading this right, the user can suppress some fields if they choose but there aren't any hidden fields they can see by changing the format?

Reply:
Yes. The user can choose to only include a couple of the fields listed, but can _only_ choose from those fields.